RocksDBStore/Store.cs

```diff
@@ -28,10 +29,7 @@ public Store(string path)
 
             // Clean all families
 
-            for (int x = 0; x <= byte.MaxValue; x++)
-            {
-                db.DropColumnFamily(x.ToString());
-            }
+            Parallel.For(0, byte.MaxValue + 1, (x) => db.DropColumnFamily(x.ToString()));
```

Comment:
I don't think that executing IO delete operations in parallel can speed up.

Reply:
I think that it should work, but what do you prefer?

Reply:
I think remove the `Parallel.For`.